docs/content/ingestion/tasks.md

```diff
@@ -122,8 +122,9 @@ compression formats. The indexSpec is optional and default parameters will be us
 |Field|Type|Description|Required|
 |-----|----|-----------|--------|
 |bitmap|Object|Compression format for bitmap indexes. Should be a JSON object; see below for options.|no (defaults to Concise)|
-|dimensionCompression|String|Compression format for dimension columns. Choose from `LZ4`, `LZF`, or `uncompressed`.|no (default == `LZ4`)|
-|metricCompression|String|Compression format for metric columns. Choose from `LZ4`, `LZF`, or `uncompressed`.|no (default == `LZ4`)|
+|dimensionCompression|String|Compression format for dimension columns. Choose from `LZ4`, `LZF`, `uncompressed`, or `none`. Note that `uncompressed` is deprecated and should only be used for backward compatibility, for newer versions, use `none` instead. |no (default == `LZ4`)|
```

Comment:
maybe it's a good idea to have a separate page talking about the metrics storage format, and move this note and the explanation there